prov/util/src/util_mr.c

```diff
@@ -152,14 +154,27 @@ static int compare_mr_keys(void *key1, void *key2)
 }
 
 
-int ofi_mr_map_init(const struct fi_provider *prov, enum fi_mr_mode mode,
+/*
+ * If a provider or app whose version is < 1.5, calls this function and passes
+ * FI_MR_UNSPEC as mode, it would be treated as FI_MR_SCALABLE.
```

Comment:
nit: please change FI_MR_SCALABLE to scalable.  The mode is mapped to 0, not the value of FI_MR_SCALABLE.